REQUIRE

```diff
@@ -4,3 +4,4 @@ Compose
 LNR
 JuliaParser
 Requires
+Compat
```

Comment:
I think it's best to go without Compat – it was only needed in one place and that's fixed anyway.